tensorboard/version.py

```diff
@@ -15,7 +15,7 @@
 
 """Contains the version string."""
 
-VERSION = "2.13.0a0"
+VERSION = "2.13.0"
```

Comment:
We want each of these changes to be separate commits in the history of the release, that's why our release guide mentions that if they're all submitted in a single PR, we should use "rebase and merge" (referring to the fact that we would have multiple commits in the same PR).

See example from a previous release: https://github.com/tensorflow/tensorboard/pull/6192/commits

So we should have a commit with only the proto changes, and separate commits for the other steps of the release process.

Also, we will probably want these proto changes in the master branch, so you can choose to either submit those changes in master, and then create this branch again, or later cherry-pick those changes into master as well.

Reply:
I didn't know about the convention of doing this all as a single PR with multiple ordered commits.